src/test/java/com/browserstack/pageobjects/SearchPage.java

```diff
@@ -4,7 +4,7 @@
 import org.openqa.selenium.WebDriver;
 
 public class SearchPage {
-    private static WebDriver webDriver;
+    private WebDriver webDriver;
```

Comment:
Didn't get why this is being changed?

Reply:
The webDriver instance was getting reused across threads.